### DISCUSSION TITLE
How to send Alpine.store('global_data') data to the server ?

### DISCUSSION THREAD

**Discussion by anjanesh** [alpinejs/alpine#3548] [1. Help] [answered]
I have a complex form which some 100+ input fields and some sections are dynamic for which the data gets shown in a table after input.
All data are in an AlpineJS global store - I want to send this data (some_other_data, cloud_servers, others) which are 2 level deep nested objects on form submit to a Django end point. 
I have a big list of 
```
document.addEventListener('alpine:init', () =>
{
    Alpine.store('global_data',
    {
        notification: '',
        some_other_data: {},
        cloud_servers: [],
        others: [],
        apps:  new Set([]),   
    });
});
```
What's the best approach - send to a `<form>` or use REST API ? And how do I consolidate the Alpine.store('global_data') and send the data to the server ?

**Comment by ekwoka** [comment, marked as answer]
What part are you having the issue with?

```js
fetch(endpoint, {
  method: 'POST',
  body: JSON.stringify(Alpine.store('global_data'))
})
```
